lib/ui/analytics/tests/analytics-spec.js

```diff
@@ -14,7 +14,7 @@ describe('avAnalytics', function() {
   availity.mock.directiveSpecHelper();
 
   beforeEach(inject(function(_avAnalytics_, $q) {
-    $el = availity.mock.compileDirective('<button data-av-analytics-on="click" data-av-analytics-action="save"></button>');
+    $el = availity.mock.compileDirective('<button data-av-analytics-on="click" data-av-analytics-category="test category" data-av-analytics-label="save"></button>');
```

Comment:
I will change it back, came to the conclusion we need a test case for Splunk and a separate test case for Piwik. The changes I made here reflect that you get the correct results for a Piwik event.